src/server/fakeServer.jsx

```diff
@@ -0,0 +1,33 @@
+import { createServer } from "miragejs";
+import { faker } from "@faker-js/faker";
+
+export default function startServer() {
+  createServer({
+    routes() {
+      this.namespace = "api";
+      this.timing = 2000;
+
+      this.get("/users/:amount", (_, request) => {
+        let amount = request.params.amount;
+
+        let x = generatePosts(amount);
+
+        let y = JSON.stringify(x);
```

Comment:
Don't use y.  Just return Json.string(x)

Reply:
haha just read this , fixed